internal/gitlab/gitlab.go

```diff
@@ -1709,3 +1709,32 @@ func TodoList(opts gitlab.ListTodosOptions, n int) ([]*gitlab.Todo, error) {
 
 	return list, nil
 }
+
+func MarkTodoAsDone(todoNum int) error {
+	_, err := lab.Todos.MarkTodoAsDone(todoNum)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func MarkAllTodosAsDone() error {
+	_, err := lab.Todos.MarkAllTodosAsDone()
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func CreateTodo(project string, mrNum int) (int, error) {
```

Comment:
We've been using the not-well-defined pattern of `<object><action>` for function names, e.g. `MRList` or `IssueCreate`, it would be better to keep the pattern here too:

`TodoMarkDone()`
`TodoMarkAllDone()`
`TodoCreate()`.

What do you think?

Reply:
I like that a lot more and ;) was wondering about the naming.  I'll definitely move to this.

Reply:
Fixed.